src/Http/Response/Format/Format.php

```diff
@@ -47,6 +54,20 @@ public function setResponse($response)
     }
 
     /**
+     * Set the formats' options.
+     *
+     * @param array $options
+     *
+     * @return void
```

Comment:
Oh, right. But `@return \Dingo\Api\Http\Response\Format\Format`, as other fluent methods in this class do.